src/utils/util/resolver-pipeline/readme.md

```diff
@@ -10,40 +10,46 @@ The sequence of XSLT transformations reflects and roughly corresponds to the ste
+| `uuid-method`             | Method for computing UUID of top-level element in output catalog. Valid values are: `random-xslt`, in which case the `random-number-generator` XPath function must be available; `random-java`, in which case the `java.util.UUID.randomUUID()` Java method must be available; `user-provided`, in which case you must specify the `top-uuid` parameter; `web-service`, referring to the `uuid-service` parameter value; and `fixed`. | `fixed`                                      |
+| `uuid-service`            | URI for a web service that produces a UUID, if `uuid-method` is `web-service`.                                                                                                                                                                                                                                                                                                                                                        | `https://www.uuidgenerator.net/api/version4` |
+| `trace`                   | If set to `on`, additional debug information will be printed to the console and `opr:ERROR`, `opr:WARNING` messages will be retained in the output document.                                                                                                                                                                                                                                                                          | None                                         |
 
-### Invoking the XSLT
+Note that URIs (addresses) given in a profile document must link correctly as absolute or relative paths to their imported catalogs, as demonstrated in [examples](../../../specifications/profile-resolution/profile-resolution-examples).
 
-Use a recent version of Saxon for best results &#8212; although we would also be *very interested* to hear from users of other XSLT engines conformant to the 3.1 family of XML standards (XSLT/XPath/XDM/XQuery).
+A serialized output of profile resolution takes the form of an OSCAL catalog. Assuming inputs are correctly formed, the output is valid to the catalog schema.
 
-The entry point for the transformation pipeline is `oscal-profile-RESOLVE.xsl`, which invokes the transformation steps in sequence, taking the source profile document as primary input. Load Saxon with your document and this stylesheet as follows (for example):
+## Invoking the Pipeline
 
-```bash
->  java -cp saxon-he-10.0.jar net.sf.saxon.Transform -t -s:YOUR_PROFILE_DOCUMENT.xml -xsl:path/to/oscal-profile-RESOLVE.xsl -o:YOUR_RESULT_BASELINE.xml
-```
+As of now, the profile resolution pipeline **is incompatible with Saxon versions newer then 10.x**. For best results, use Saxon 10, though you may have luck with other XSLT engines conformant to the 3.1 family of XML standards (XSLT/XPath/XDM/XQuery).
```

Comment:
Blocking: if we are going to merge the resolver fix in #1639 (heads up, I definitely intend to ASAP), I would rather not leave documentation we will need to update right after.

```suggestion
The profile resolver and other utilities in this repo rely on open-source libraries and utilities developed by third parties. To review the software version used by this project, please review [the `build` directory for manifests of software and the specific versions](../../../../build) we recommend for running utilities from this repo, specifically the recommended [version of Saxon XSLT processor defined in the pom.xml manifest](../../../../build/pom.xml).
```

Reply:
@aj-stein-nist If this is the case I will also add the wrapper script to this commit along with updating the readme. Please hold before merging.